tests/test_odcs_util.py

```diff
@@ -94,7 +94,13 @@ def compose_json(state, state_name, source_type='module', source=MODULE_NSV,
     ['no_deps'],
     ['breakfast', 'lunch'],
 ))
-def test_create_compose(odcs_client, source, source_type, packages, sigkeys, arches, flags):
+@pytest.mark.parametrize(('multilib_arches', 'multilib_method'), (
+    (None, None),
+    (['x86_64'], ['all']),
+    (['breakfast', 'lunch'], ['some', 'random'])
```

Comment:
Can we have a test that sets multilib_arches to some non-empty list but multilib_method to something falsy?